test/check-machines-networks

```diff
@@ -425,6 +425,36 @@ class TestMachinesNetworks(machineslib.VirtualMachinesCase):
             xfail_error="Invalid",
         ).execute()
 
+        # Check "prefix must be multiple of 8"
+        NetworkCreateDialog(
+            self,
+            name="test_network",
+            forward_mode="open",
+            ip_conf="IPv4 only",
+            ipv4_address="192.168.100.1",
+            ipv4_netmask="26",
+            ipv4_dhcp_start="300.2.1",
+            ipv4_dhcp_end="168..1.1.1",
```

Comment:
Can you  please make these valid? It's a bit confusing to have more than one wrong value per test case. These are already covered by the previous test.

Reply:
This does in fact test something useful: As long as the prefix is invalid, we don't want any fields to be checked that depend on a valid prefix. Thus, this checks that the DHCP fields do _not_ have errors.

This could be improved, I guess: We can call out syntax errors even without a valid prefix. (The prefix is only used to check whether the values are out of bound.)

Not sure how far we want to go...

I make these valid.